content/docs/user-guide/scripting/lua/lua-editor.md

```diff
@@ -1,6 +1,70 @@
 ---
-title: Lua Editor
-description: ' Learn about the Lua Editor in Open 3D Engine. '
+linkTitle: Lua Editor
+title: Introduction to Open 3D Engine's Lua Editor
+description: Learn about the Lua Editor in Open 3D Engine.
+toc: true
+weight: 300
 ---
 
-The Open 3D Engine Lua Editor (Lua IDE) offers an intuitive integrated development environment (IDE) that makes it easy to author, debug, and edit Lua scripts when you create or extend your game. Lua Editor is a standalone application, but can be opened directly from the Open 3D Engine Editor using the Edit Mode Toolbar.
+The O3DE **Lua Editor** (Lua IDE) offers an intuitive integrated development environment (IDE) that makes it easy to author, debug, and edit Lua scripts when you create or extend your game. Lua Editor is a standalone application, but can be opened directly from the **O3DE Editor** using the Edit Mode Toolbar.
+
+## Editing 
+
+Lua Editor can open multiple scripts at the same time. Each script has its own tab in the editor. The editor provides a standard set of capabilities for text editing but also includes useful features for editing source code.
+
+The following table summarizes the options available while editing and debugging.
+
+| **Action** | **Keyboard Shortcut** |
+| --- | --- |
+| Comment selected block | Ctrl+K |
+| Copy | Ctrl+C |
+| Cut | Ctrl+X |
+| Find | Ctrl+F |
+| Find in open files | Ctrl+Shift+F |
+| Find next | F3 |
+| Fold source functions | Alt+0 |
+| Go to line | Ctrl+G |
+| Paste | Ctrl+V |
+| Quick find local | Ctrl+F3 |
+| Quick find local reverse | Ctrl+Shift+F3 |
+| Redo | Ctrl+Y |
+| Replace | Ctrl+R |
+| Replace in open files | Ctrl+Shift+R |
+| Select all | Ctrl+A |
+|  Select to brace¹  | Ctrl+Shift+\] |
+| Transpose lines down | Ctrl+Shift+Down Arrow |
+| Transpose lines up | Ctrl+Shift+Up Arrow |
+| Uncomment selected block | Ctrl+Shift+K |
+| Undo | Ctrl+Z |
+| Unfold source functions | Alt+Shift+0 |
```

Comment:
The style guide calls for bold input combinations. I can see an argument for not using bold as the table already highlights them, but for consistency... Also, is the Select to Brace hotkey correct?

```suggestion
| **Action** | **Keyboard Shortcut** |
| --- | --- |
| Comment selected block | **Ctrl+K** |
| Copy | **Ctrl+C** |
| Cut | **Ctrl+X** |
| Find | **Ctrl+F** |
| Find in open files | **Ctrl+Shift+F** |
| Find next | **F3** |
| Fold source functions | **Alt+0** |
| Go to line | **Ctrl+G** |
| Paste | **Ctrl+V** |
| Quick find local | **Ctrl+F3** |
| Quick find local reverse | **Ctrl+Shift+F3** |
| Redo | **Ctrl+Y** |
| Replace | **Ctrl+R** |
| Replace in open files | **Ctrl+Shift+R** |
| Select all | **Ctrl+A** |
| Select to brace¹ | **Ctrl+Shift+\]** |
| Transpose lines down | **Ctrl+Shift+Down Arrow** |
| Transpose lines up | **Ctrl+Shift+Up Arrow** |
| Uncomment selected block | **Ctrl+Shift+K** |
| Undo | **Ctrl+Z** |
| Unfold source functions | **Alt+Shift+0** |
```

Reply:
It is **Ctrl+Shift+]**, I will edit.